discord-bot/es-ES.jsonc

```diff
@@ -345,64 +345,64 @@
   "default_currency": "usd",
 
   // Did this bot save you some money? Consider [donating]({url}) to keep it alive. Thanks!
-  "donation_notice": "¿Éste bot te ha ayudado a ahorrar algo de dinero? Si es así, considera [donarnos]({url}) para mantenerlo con vida. ¡Muchas gracias!",
+  "donation_notice": "¿Este bot te ha ayudado a ahorrar algo de dinero? Si es así, considera [donar]({url}) para mantenerlo con vida. ¡Muchas gracias!",
 
   // Claim All
-  "expressbutton_label": "Reclamar Todo",
+  "expressbutton_label": "Reclamar todo",
 
   // {next}
-  "fluid_list_delimiter_first": "",
+  "fluid_list_delimiter_first": "{next}",
 
   // {prev}, and {next}
-  "fluid_list_delimiter_last": "",
+  "fluid_list_delimiter_last": "{prev}, y {next}",
 
   // {prev}, {next}
-  "fluid_list_delimiter_middle": "",
+  "fluid_list_delimiter_middle": "{prev}, {next}",
 
   // {prev} and {next}
-  "fluid_list_delimiter_only": "",
+  "fluid_list_delimiter_only": "{prev} y {next}",
 
   // ONLY ON APRIL FIRST
-  "free_recommendation_april_fools": "",
+  "free_recommendation_april_fools": "SOLO EL 1 DE ABRIL",
 
   // Free on {store}!
-  "free_recommendation_by_store": "",
+  "free_recommendation_by_store": "¡Gratis en {store}!",
 
   // Interested in {tag}?
-  "free_recommendation_by_tag": "",
+  "free_recommendation_by_tag": "¿Interesado en {tag}?",
 
   // Expires in {hours, plural, =1 {1 hour} other {# hours} }!
-  "free_recommendation_expires_hours": "",
+  "free_recommendation_expires_hours": "¡Expira en {hours, plural, =1 {1 hour} other {# hours} }!",
```

Comment:
Hola, just for your information (and this is fairly advanced so dont worry) but you can change out the `1 hour` and `# hours` as well here :D
```suggestion
  "free_recommendation_expires_hours": "¡Expira en {hours, plural, =1 {1 hora} other {# horas}}!",
```
You can use an online tool like this to check your input: https://format-message.github.io/icu-message-format-for-translators/editor.html

<img width="1073" height="855" alt="Image" src="https://github.com/user-attachments/assets/0e43c078-dc5b-4bfe-92d9-a3293ba6ccb7" />